aws/table_aws_ecr_repository.go

```diff
@@ -108,12 +108,6 @@ func tableAwsEcrRepository(_ context.Context) *plugin.Table {
 				Hydrate:     getAwsEcrRepositoryLifecyclePolicy,
 				Type:        proto.ColumnType_TIMESTAMP,
 			},
-			{
```

Comment:
Was this removed from the API response?

Reply:
Typically, we don't include the `max_results` column in tables, as it's only used for pagination operations on the API.